syft/generic/pointers/object_pointer.py

```diff
@@ -1,18 +1,19 @@
-import syft
-from syft.frameworks.torch.tensors.interpreters import abstract
-from syft import messaging
-from syft import exceptions
-
 from typing import List
 from typing import Union
 from typing import TYPE_CHECKING
 
+import syft
+from syft import exceptions
+from syft import messaging
+from syft.codes import MSGTYPE
```

Comment:
Should this be refactored to use the new message class

Reply:
hm, probably! thanks for the ping

Reply:
actually, this should've been done elsewhere and is beyond the scope of this PR, so I'll just open an issue for it